src/99-appendices/03-hed.md

```diff
@@ -185,3 +185,39 @@ any present HED information will be validated using the latest version of the HE
 which is bound to result in problems.
 Hence, it is strongly RECOMMENDED that the `HEDVersion` field be included when using HED
 in a BIDS dataset.
+
+### Using HED library schemas
+
+HED also allows you to use one or more specialized vocabularies along with
+the base vocabulary. These specialized vocabularies are developed by
+communities of users and are available in the GitHub
+[hed-schema-library](https://github.com/hed-standard/hed-schema-library) repository.
+
+Example: The following `dataset_description.json` file specifies that the
+[`HED8.1.0.xml`](https://github.com/hed-standard/hed-specification/tree/master/hedxml/HED8.1.0.xml)
+base schema should be used along with the
+SCORE library for clinical neurological annotation and a test library
+located at [HED_score_0.0.1.xml](https://github.com/hed-standard/hed-schema-library/blob/main/library_schemas/score/hedxml/HED_score_0.0.1.xml) and [HED_testlib_1.0.2.xml](https://github.com/hed-standard/hed-schema-library/blob/main/library_schemas/testlib/hedxml/HED_testlib_1.0.2.xml), respectively.
+
+```JSON
+{
+  "Name": "A great experiment",
+  "BIDSVersion": "1.7.0",
+  "HEDVersion": {
+      "base": "8.1.0",
+      "libraries": {
+          "sc": "score_0.0.1",
+          "ts": "testlib_1.0.2"
+      }
+  }
+}
+```
+The `sc:` and `ts:` are user-chosen prefixes used to distinguish the sources
+of the terms in the HED annotation. In the following HED annotation:
```

Comment:
what rules are there for user-chosen prefixes? Are there reserved keywords that MUST NOT be used? Are these supposed to be alphanumeric? Or just lower case letters a to z?

Reply:
There are no restrictions on what to use for a user-chosen prefix -- just that it be alphanumeric. It can even be an existing HED tag, although we recommend that it be short for readability.

The prefix does not have to be lowercase.  The examples use lower case because by style convention, HED tags start with an upper case letter, and the lower case prefix makes the tag string easier to read.

Reply:
Okay, that makes sense -- I would encode this rule "user-chosen prefixes MUST be alphanumeric" into both the spec text and the schema

Reply:
Let us know your thoughts on our SCORE schema library [here](https://github.com/hed-standard/hed-schema-library/issues/24). And a bids example with HED annotations using the SCORE schema library is coming soon!